portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,13 @@ function loadSkills () {
  */
  async function fetchFromData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('secret-message').innerText = quote; 
+  // This gives a list of messages
+  const msgs = await response.json();
+  console.log(msgs); 
+
+  // Adds all of the messages to an unordered list
```

Comment:
This does not match the code: instead of an unordered list (`<ul>` tag), you are adding `<p>` tag.

Reply:
OH yea I'm so sorry I made it a ul at first and then changed it into a div instead. updated the comments.